problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,17 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
+function getCountriesSortedByPopulation(arr) {
+    return arr.sort((country1, country2) => {
+        console.log(arr);
```

Comment:
you're close. Make sure to consult with the MDN documentation to ensure you can sort objects. You'll want to do something like country1.population - country2.population